packages/app/components/settings/tabs/email.tsx

```diff
@@ -64,17 +53,20 @@ export const EmailTab = ({ index = 0 }: EmailTabProps) => {
           )
         }
       />
-      {emailWallets?.length === 0
-        ? ListEmptyComponent()
-        : emailWallets?.map((item) => (
-            <View key={item.address}>
-              <SettingsEmailSlot
-                email={item.email}
-                address={item.backendAddress}
-              />
-              <SlotSeparator />
-            </View>
-          ))}
+      {emailWallets?.length === 0 ? (
+        <EmptyPlaceholder
```

Comment:
we should use `EmptyPlaceholder` uniformly to handle empty list instead of any custom empty component, which will  help clean up the code